easybuild/easyblocks/i/imkl.py

```diff
@@ -357,7 +357,7 @@ def sanity_check_step(self):
             if self.cfg['m32']:
                 raise EasyBuildError("Sanity check for 32-bit not implemented yet for IMKL v%s (>= 10.3)", self.version)
             else:
-                mkldirs = ["bin", "mkl/bin", "mkl/bin/intel64", "mkl/lib/intel64", "mkl/include"]
+                mkldirs = ["bin", "mkl/bin", "mkl/lib/intel64", "mkl/include"]
```

Comment:
we should only remove the check on `mkl/bin/intel64` for the newest versions of `imkl`, and also check what kind of potential fallout this has (which tools are located in `mkl/bin/intel`, and where are they used by EB)?